packages/drivers/odsp-driver/src/exposeJoinSessionInfo.ts

```diff
@@ -0,0 +1,26 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IResolvedUrl } from "@fluidframework/driver-definitions";
+import { ISocketStorageDiscovery } from "./contractsPublic";
+import { getJoinSessionCacheKey, getOdspResolvedUrl } from "./odspUtils";
+import { OdspDocumentServiceFactory } from "./odspDocumentServiceFactory";
+
+/**
+ * Api which returns the current join session response stored in non persistent cache, if present
```

Comment:
While this description is good, I think it's hard to parse by users (who have no knowledge of the internals of the driver). I'd add here a bit more using terminology that is more targeted to users of this API.
I.e., this API would return info about relay service session only if this factory established (or attempted to establish) connection very recently. Otherwise, it will return undefined.